README.md

```diff
@@ -37,6 +37,7 @@ The following documents are available:
 | Kafka Protocol Binding        | [v1.0](https://github.com/cloudevents/spec/blob/v1.0/kafka-protocol-binding.md) | [master](https://github.com/cloudevents/spec/blob/master/kafka-protocol-binding.md) |
 | MQTT Protocol Binding         | [v1.0](https://github.com/cloudevents/spec/blob/v1.0/mqtt-protocol-binding.md) | [master](https://github.com/cloudevents/spec/blob/master/mqtt-protocol-binding.md)  |
 | NATS Protocol Binding         | [v1.0](https://github.com/cloudevents/spec/blob/v1.0/nats-protocol-binding.md) | [master](https://github.com/cloudevents/spec/blob/master/nats-protocol-binding.md)  |
+| Pub/Sub Protocol Binding      | [v1.0](https://github.com/cloudevents/spec/blob/v1.0/pubsub-protocol-binding.md) | [master](https://github.com/cloudevents/spec/blob/master/pubsub-protocol-binding.md)  |
```

Comment:
Google Cloud Pub/Sub?   As-is this naming seems too generic.

Reply:
removing these links for now as the markup linter complaints... 
I guess we should add the links in a follow up PR